tests/test_files/bundles/nest/bundle2/svcat.package.yaml

```diff
@@ -0,0 +1,5 @@
+packageName: svcat
+channels:
+- name: alpha
+  currentCSV: svcat.v0.1.34
+      
```

Comment:
This is missing a newline at the end of the file.

Reply:
Ah, I left some whitespace on line 5, will fix this. Thank you!